pcweb/pages/docs/tutorial/adding_state.py

```diff
@@ -236,34 +285,28 @@ def adding_state():
             doclink("events docs", events.path),
             ".",
         ),
-        # subheader("Clearing the Input"),
-        # doctext(
-        #     "Currently the input doesn't clear after the user clicks the button. ",
-        #     "We can fix this by adding an ",
-        #     rx.code("id"),
-        #     " to the input and creating an event to clear it. ",
-        # ),
-        # docdemobox(code_out2),
-        # doccode(code2),
-        # doccode(state2),
-        # doctext(
-        #     "After setting the question, we return an event from our ",
-        #     rx.code("answer"),
-        #     " event handler to clear the input. ",
-        #     "Learn more about returning events in the ",
-        #     doclink("events docs", f"{events.path}#returning-events-from-event-handlers"),
-        #     ".",
-        # ),
-        # doctext(
-        #     "We use the ",
-        #     rx.code("set_value"),
-        #     " function to set the value of the ",
-        #     rx.code("question"),
-        #     " state var to an empty string. ",
-        #     "See the ",
-        #     doclink("special events docs", special_events.path),
-        #     " for more info on special events. ",
-        # ),
+        subheader("Clearing the Input"),
+        doctext(
+            "Currently the input doesn't clear after the user clicks the button. ",
+            "We can fix this by binding the value of the input to ",
+            rx.code("question"),
+            " and creating an event ",
+            rx.code("clear_question"),
+            " to clear it. ",
+        ),
+        docdemobox(code_out2),
+        doccode(code2),
+        doccode(state2),
+        doctext(
+            "After setting the question, we return an event from our ",
```

Comment:
I think we should modify this to
```python
def answer(self):
    # Our chatbot is not very smart right now...
    answer = "I don't know!"
    self.chat_history.append((self.question, answer))
    self.question = ""
```
Then we can get rid of the second event handler. I had it that way in the commented code because this method wasn't working at the time.